examples/liberty-car-booking/src/main/liberty/config/server.xml

```diff
@@ -0,0 +1,31 @@
+<server description="Sample Liberty server">
+    <featureManager>
+    	<!-- Eclipse Microprofile 6.1 -->
+    	<!-- See details here: https://openliberty.io/docs/ref/feature/#microProfile-6.1.html -->
+    	<feature>microProfile-6.1</feature>
+    </featureManager>
+
+    <variable name="default.http.port" defaultValue="9080"/>
+    <variable name="default.https.port" defaultValue="9443"/>
+
+    <webApplication location="${project.name}.war" contextRoot="/">
+    	<appProperties>
+     		<property name="app.docs-for-rag.dir" value="${docragdir}"/>
+  		</appProperties>
+  		<classloader commonLibraryRef="djLNativeLib" /> 
+    </webApplication>
+    
+    <library id="djLNativeLib">
+    	<fileset dir="${shared.resource.dir}/lib/tokenizers/" includes="*.dll, *.so" />
+	</library>
+    <mpMetrics authentication="false"/>
+    
+    <applicationMonitor dropinsEnabled="false"/>
+
+    <logging traceSpecification="com.ibm.ws.microprofile.health.*=all" />
```

Comment:
I don't think you need to collect this MP Health traces.

Reply:
Done!